WebContent/js/components/Setting.js

```diff
@@ -0,0 +1,117 @@
+    color: #39464e;
+    fontFamily: "Roboto", "Helvetica", "Arial", sans-serif;
+`;
+const H3 = styled.h3`
+    border-top-color:rgb(233, 233, 233);
+    border-top-style:solid;
+    border-top-width:1px;
+    color:rgb(57, 70, 78);
+    font-size:13px;
+    font-weight:700;
+    padding-top:8px;
+`;
+
+const SettingSection = styled.div`
+    width: 100%;
+    display: grid;
+    grid-template-columns: 100%
+`;
+
+export default class SettingForm extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            enableReduxLogger: getStorageItem(ENABLE_REDUX_LOGGER) || false,
+            notificationDuration: getStorageItem(NOTIFICATION_DURATION) || 5000,
+        };
+
+        this.mapStorageKey = new Map();
+        this.mapStorageKey.set(ENABLE_REDUX_LOGGER, 'enableReduxLogger');
+        this.mapStorageKey.set(NOTIFICATION_DURATION, 'notificationDuration');
+
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange = ev => {
+        let value = ev.target.value;
+        const key = this.mapStorageKey.get(ev.target.name);
+
+        if ('checked' in ev.target) {
+            value = ev.target.checked;
+        }
+
+        this.setState({
+            [key]: value,
+        });
+
+        setStorageItem(ev.target.name, value);
+    }
+
+    render() {
+        const { notificationDuration, enableReduxLogger } = this.state;
+        return (
+            <Settings>
+                <h3>Preferences</h3>
+                <form>
+                    <H3>App</H3>
+                    <SettingSection>
+                        <FormControl>
+                            <TextField
+                                select={true}
+                                label="*Notification Duration"
+                                value={notificationDuration.toString()}
+                                onChange={this.handleChange}
+                                name={NOTIFICATION_DURATION}
+                            >
+                                <MenuItem id="notification-small" key="small" value="5000" >Small</MenuItem>
+                                <MenuItem id="notification-medium" key="medium" value="10000" >Medium</MenuItem>
+                                <MenuItem id="notification-large" key="large" value="15000" >Large</MenuItem>
+                            </TextField>
+
+
+                        </FormControl>
+                    </SettingSection>
+                    <H3>Development</H3>
+                    <SettingSection>
+                        <FormControl>
+                            <FormControlLabel
+                                control={<Switch
+                                    name={ENABLE_REDUX_LOGGER}
+                                    checked={enableReduxLogger}
+                                    onChange={this.handleChange}
+                                />}
+                                label="*Redux Logger"
```

Comment:
I think this label should be changed to "Browser console logging", Redux may confuse users

Reply:
done